src/Umbraco.Commerce.PaymentProviders.Mollie/UriExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Specialized;
+using System.Web;
+
+namespace Umbraco.Commerce.PaymentProviders.Mollie
+{
+    public static class UriExtensions
```

Comment:
Lets make this internal. Use `InternalsVisibleTo` if you need access from your test project https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-8.0

Reply:
Oops. Thanks for pointing it out. This file is completely redundant because I switched to `Microsoft.AspNetCore.WebUtilities` package. I removed it now.